tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -111,18 +112,18 @@ export function getNpmPackageName(packageRoot: string): string {
 
 export function getApiReviewPath(packageRoot: string): string {
     const sdkType = getSDKType(packageRoot);
-    const reviewDir = path.join(packageRoot, 'review');
+    const npmPackageName = getNpmPackageName(packageRoot);
```

Comment:
Can you revert this function?

Reply:
if use https://github.com/Azure/azure-sdk-tools/pull/10499/files#diff-9c7401a3c34bb2f6aa6dc3d2a2f2bacb068b362f8becda4c16dcb2dfddd56065L125, it will fail the apiview path read if it the npm package doesn't have review folder